coalib/bearlib/__init__.py

```diff
@@ -82,34 +100,56 @@ def _deprecate_decorator(func):
         logged_deprecated_args = set()
 
         def wrapping_function(*args, **kwargs):
-            for arg, depr_arg_and_modifier in depr_args.items():
-                deprecated_arg, _func = (
```

Comment:
I dont see why you need to modify this line.

`deprecated_arg` will contain a dict, but that can be fixed later.